### PR TITLE
Nemo extreme mvp pages

### PR DESCRIPTION
**Pull request by sjpaige**
Re-opening this branch with the amplify server. There are quite a few new components and pages undergoing changes.

### CODE REVIEW

File: src/Pages/VolunteerPage.jsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+              <Typography variant="body1">
+                If you want further information and want to have a chat, don’t
+                hesitate to contact us though our form.
+              </Typography>
+            </Box>
+          </Grid>
+        </Grid>
+        <Box className={classes.backgroundSecondary}>
+          <Typography className={classes.rolesHeading} variant="h2">
+            Open Roles
```
Comment (stuttskl) [inline src/Pages/VolunteerPage.jsx:137]: Can we add or adjust the padding here? 
Screen Shot 2022-01-12 at 11 45 36 AM

**Review by stuttskl — APPROVED** [review]
Approved pending small fix mentioned in comments

File: src/Pages/VolunteerPage.jsx (hunk 1 above)
Comment (sjpaige) [reply src/Pages/VolunteerPage.jsx:137]: Definitely.

Comment (sjpaige) [reply src/Pages/VolunteerPage.jsx:136]: image
How is this?

**Pull request merged**